### ISSUE TITLE
Support SAM file input through stdin to reduce I/O

### ISSUE THREAD

**Issue by ElDeveloper** [qiyunzhu/woltka#162]
For cloud-based workflows, reducing the read/write steps in a pipeline impacts the cost and efficiency of the workflow. Is there a way for `woltka classify` to read data from `stdin` so pipelines that QC sequences could directly pipe into `bowtie2` for alignment and into `woltka` for classification? For example something like:

```
fastp [READ raw sequences] | minimap2 ... | seqtk .... | bowtie2 ... | woltka classify .... [WRITE alignment file and tables]
```

This way the only reading operation related to the sequences would happen with `fastp` and the only writing operation related to the sequences/alignment would happen with `woltka`.

**Comment by qiyunzhu** [comment]
@ElDeveloper This is a great suggestion. It has already been implemented (#155 ) and documented ([here](https://github.com/qiyunzhu/woltka/blob/master/doc/input.md#input-filepath) and [here](https://github.com/qiyunzhu/woltka/blob/master/doc/faq.md#does-woltka-support-bam-and-cram-formats)). Examples:

```bash
bowtie2 -x db -U input.fq | woltka classify -i - -o output.biom
```
```bash
samtools view input.bam | woltka classify -i - -o output.biom
```

**Comment by ElDeveloper** [comment]
That's wonderful, looks like I completely missed that. This will help a lot for sure!

**Comment by qiyunzhu** [comment]
@ElDeveloper Was only implemented in the most recent release (v0.1.4) :)
